frontend/app/views/spree/users/show.html.erb

```diff
@@ -13,7 +13,7 @@
 
       <div class="d-lg-inline-block account-page-user-info-item-store-credits">
         <p class="my-0 text-uppercase text-center">
-          <%= Spree.t(:store_credit_name) %>: <b class="account-page-user-info-item-store-credits-amount"><%= Spree::Money.new(@user.total_available_store_credit).to_s.gsub('.00', '') %></b>
+          <%= Spree.t(:store_credit_name) %>: <b class="account-page-user-info-item-store-credits-amount"><%= Spree::Money.new(@user.total_available_store_credit, currency: current_currency).to_s.gsub('.00', '') %></b>
```

Comment:
@aplegatt actually it would be better to modify

https://github.com/spark-solutions/spree/blob/ce560266090ed42363eeb76d664aa1027b3bb811/core/app/models/concerns/spree/user_methods.rb#L58

to

```ruby
def total_available_store_credit(currency = nil)
  currency ||= Spree::Config[:currency]
  store_credits.where(currency: currency).reload.to_a.sum(&:amount_remaining)
end
```

and use it here:

```suggestion
          <%= Spree.t(:store_credit_name) %>: <b class="account-page-user-info-item-store-credits-amount"><%= Spree::Money.new(@user.total_available_store_credit(current_currency)).to_s.gsub('.00', '') %></b>
```

Reply:
`total_available_store_credit` is returning only amount of store credits based on currency but currency symbol is still managed by `Spree::Money` and is not correctly displayed without passing current store currency.
<img width="386" alt="Screenshot 2020-06-09 09 31 01" src="https://user-images.githubusercontent.com/5177038/84118948-fc3afe00-aa33-11ea-9be3-f2f492e3ab6a.png">